hellogsm-web/src/main/java/team/themoment/hellogsm/web/domain/identity/service/AuthenticateCodeService.java

```diff
@@ -1,7 +1,18 @@
 package team.themoment.hellogsm.web.domain.identity.service;
 
+import team.themoment.hellogsm.web.domain.identity.domain.AuthenticationCode;
 import team.themoment.hellogsm.web.domain.identity.dto.request.AuthenticateCodeReqDto;
 
+/**
+ * AuthenticationCode를 인증하는 인터페이스입니다.
+ * @see AuthenticationCode 참고해주세요.
```

Comment:
일반적으로 `@see` 태그의 경우, 부연설명을 하지 않고 단순히 참고할 요소만 명시하는 편입니다. 수정하는게 좋겠습니다.
`@see` 태그를 사용하는 다른 부분도 수정해주세요.

```suggestion
 * @see AuthenticationCode
```

Reply:
부연설명 제거 하였습니다